src/common/hdfs/test/HdfsHelperTest.cpp

```diff
@@ -12,15 +12,15 @@ namespace nebula {
 namespace hdfs {
 
 TEST(HdfsHelper, ls) {
-    auto helper = HdfsCommandHelper();
-    auto result = helper.ls("127.0.0.1", 9000, "/");
-    ASSERT_TRUE(result.ok()) << result.status();
+//    auto helper = HdfsCommandHelper();
```

Comment:
why cancel this test case ?

Reply:
It should never pass.

Reply:
maybe you add some check about the result code Could you remove it?

Reply:
In fact, it should fail if can not connect to hdfs

Reply:
Please remove the code if useless.

Reply:
Please fix the comment